app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -193,7 +200,7 @@ def reenter_phone_number_path
     end
   end
 
-  def presenter_for(otp_code_method, data_model)
+  def presenter_for(otp_code_method, data_model = otp_phone_view_data)
```

Comment:
since we are always using the default, should we just remove this arg altogether and change references to `data_model` to be `otp_phone_view_data` ?